ManageTeamsIdentityMobileAndDesktop/src/main/java/com/communication/quickstart/App.java

```diff
@@ -0,0 +1,62 @@
+package com.communication.quickstart;
+
+import com.azure.communication.identity.CommunicationIdentityClient;
+import com.azure.communication.identity.CommunicationIdentityClientBuilder;
+import com.azure.communication.identity.models.GetTokenForTeamsUserOptions;
+import com.azure.core.credential.AccessToken;
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import com.microsoft.aad.msal4j.InteractiveRequestParameters;
+import com.microsoft.aad.msal4j.PublicClientApplication;
+
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Set;
+
+public class App {
+
+    public static void main( String[] args ) throws Exception {
+        System.out.println("Azure Communication Services - Communication access token Quickstart");
+        // Quickstart code goes here
+
+        // You need to provide your Azure AD client ID and tenant ID
+        String appId = "<contoso_application_id>";
+        String tenantId = "<contoso_tenant_id>";
+        String authority = "https://login.microsoftonline.com/" + tenantId;
+
+        // Create an instance of PublicClientApplication
+        PublicClientApplication pca = PublicClientApplication.builder(appId)
+                .authority(authority)
+                .build();
+
+        String redirectUri = "http://localhost";
+        Set<String> scope = new HashSet<String>();
+        scope.add("https://auth.msft.communication.azure.com/Teams.ManageCalls");
+        scope.add("https://auth.msft.communication.azure.com/Teams.ManageChats");
+
+        // Create an instance of InteractiveRequestParameters for acquiring the AAD token and object ID of a Teams user
+        InteractiveRequestParameters parameters = InteractiveRequestParameters
+                .builder(new URI(redirectUri))
+                .scopes(scope)
+                .build();
+
+        // Retrieve the AAD token and object ID of a Teams user
+        IAuthenticationResult result = pca.acquireToken(parameters).get();
+        String teamsUserAadToken = result.accessToken();
+        String[] accountIds = result.account().homeAccountId().split("\\.");
+        String userObjectId = accountIds[0];
```

Comment:
is there no better way to get the userId?

Reply:
In the IAuthenticationResult it was not exposed as a separate field

Reply:
this? https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/cdff1cb77170569860cb39eae6dda56a93e3da2d/src/main/java/com/microsoft/aad/msal4j/UserIdentifier.java

Reply:
nope, we are getting this https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/cdff1cb77170569860cb39eae6dda56a93e3da2d/src/main/java/com/microsoft/aad/msal4j/IAccount.java

Reply:
` public static UserIdentifier fromHomeAccountId(String homeAccountId) {`

this wouldn't work?

Reply:
Oh I got your point, yeah I can use that method

Reply:
yes, as a parameter fro CTE we should provide oid in string format but oid is private field in UserIdentifier class